src/main/java/org/opentripplanner/netex/loader/mapping/StopTimesMapper.java

```diff
@@ -78,6 +78,11 @@ MappedStopTimes mapToStopTimes(
 
             StopPointInJourneyPattern stopPoint = findStopPoint(pointInJourneyPattern, journeyPattern);
             Stop stop = lookupStop(stopPoint, quayIdByStopPointRef, stopsById);
+            if (stop == null) {
+                LOG.warn("Stop not found for {}. Trip {} will not be mapped.",
+                    stopPoint != null ? stopPoint.getId() : "null", trip.getId());
```

Comment:
This only log the id of the stopPoint, maybe also log the missing stop id:  `stopPoint.getScheduledStopPointRef().getValue().getRef()`? 

An while at it, the `Stop not found for {}` is missing the context of the id, e.g.: `Stop not found for stop-point in journey pattern {}`?